packages/nodejs/.changesets/add-senderror-helper-to-tracer.md

```diff
@@ -0,0 +1,11 @@
+---
+bump: "patch"
+---
+
+Add sendError helper to Tracer object.
+
+This new helper allows you to track an error isolatedly without a span present in
+the context.
+
+An error is passed and a callback with a span is provided to add desired metadata
+to it.
```

Comment:
Making a suggestion with a code block is weird and doesn't render really well. But the last code block is a JavaScript block.

```suggestion
Add sendError helper to Tracer object.

This new helper allows you to track an error separately from any other span inside the current context. Or use it to set up in your own error handling to report errors in a catch-statement if no performance monitoring is needed.

```js
try {
  // Do complex stuff
} catch (error) {
  appsignal.tracer().setError(error, function(span) {
    span.setName("daily.task"); // Set a recognizable action name
    span.set("user_id", user_id); // Set custom tags
  });
}
```
```